modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -37,7 +46,11 @@ export interface ComboboxProps extends GrowthBehavior, React.HTMLAttributes<HTML
   /**
    * The autocomplete items of the Combobox. This array of menu items is shown under the text input.
    */
-  autocompleteItems?: React.ReactElement<MenuItemProps>[];
+  autocompleteItems?: React.ReactElement<ComboBoxMenuItemProps>[];
+  /**
+   * The autocomplete items of the Combobox. This prop takes precedent over autocompleteItems and displays the items with a header.
```

Comment:
I don't love the idea of ignoring one prop if another is provided. Could we just make `autocompleteItems` accept `React.ReactElement<ComboBoxMenuItemProps>[] || ComboBoxMenuItemGroup[]` and do a check for one vs. the other?